infrastructure/terraform/components/reporting/module_powerbi_gateway_vpc.tf

```diff
@@ -0,0 +1,83 @@
+module "powerbi_gateway_vpc" {
+
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "5.5.1"  # Adjust to the latest version
+
+  create_vpc = var.enable_powerbi_gateway
+
+  name = "${local.csi}-powerbi-gateway-vpc"
+  cidr = "10.0.0.0/16"
+
+  azs             = var.availability_zones
+  public_subnets  = var.public_subnet_cidrs
+  private_subnets = var.private_subnet_cidrs
+
+  enable_nat_gateway = true
+  single_nat_gateway = false
+  enable_dns_support = true
+  enable_dns_hostnames = true
+  create_igw = true
+
+  tags = {
+    Name = "${local.csi}-powerbi-gateway-vpc"
+  }
+
+  public_subnet_tags = {
+    Name = "${local.csi}-powerbi-gateway-public-subnet"
+  }
+
+  private_subnet_tags = {
+    Name = "${local.csi}-powerbi-gateway-private-subnet"
+  }
+
+  nat_gateway_tags = {
+    Name = "${local.csi}-powerbi-gateway-ngw"
+  }
+
+  public_route_table_tags = {
+    Name = "${local.csi}-powerbi-gateway-public-route-table"
+  }
+
+  private_route_table_tags = {
+    Name = "${local.csi}-powerbi-gateway-private-route-table"
+  }
+}
+
+resource "aws_security_group" "powerbi_gateway" {
+  count = var.enable_powerbi_gateway ? 1 : 0
+
+  name   = "${local.csi}-powerbi-gateway-security-group"
+  vpc_id = module.powerbi_gateway_vpc.vpc_id
+
+  egress {
+    from_port   = 80
+    to_port     = 80
```

Comment:
can we see if we can go without port 80, would be surprising if any calls go over http and not https

Reply:
https://learn.microsoft.com/en-us/data-integration/gateway/service-gateway-communication